lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -114,8 +139,33 @@ def test_create_bad(self, monkeypatch, server_config, make_logger, tarball):
         )
         assert exc.value.dataset == tarball.name
 
+    def test_duplicate(self, monkeypatch, server_config, make_logger, tarball):
+        """
+        Test behavior when we try to create a new dataset but the tarball file
+        name already exists
+        """
+        monkeypatch.setattr("selinux.is_selinux_enabled", lambda a: 0)
```

Comment:
I don't think that `is_selinux_enabled()` takes any parameters, so I think the second argument here should be `lambda: 0`.  (The analogous comment applies to lines 168 and 222.)

Reply:
"I'm *not* a method." ☑️